tools/run-dev.py

```diff
@@ -147,7 +147,8 @@
 if not os.path.exists("node_modules/.bin/handlebars") and os.path.exists("node_modules/handlebars"):
     print("Handlebars binary missing due to rebase past .gitignore fixup; fixing...")
     subprocess.check_call(["rm", "-rf", "node_modules/handlebars"])
-    subprocess.check_call(["yarn", "install", "--non-interactive", "--prefer-offline"])
+    YARN_BIN = "/srv/zulip-npm-cache/yarn/bin/yarn"
```

Comment:
The issue was actually with `Git`; if you rebase past a certain commit that's now in the far past, it'd delete part (but not all) of your handlebars checkout in a most obnoxious fashion.  

Regardless, I think this whole block can be removed.  I'll take care of it now and merge to master now, and then you can rebase; then I can write a proper commit message for it.